wrappers/python/nntile/nntile_core/tile.pyi

```diff
@@ -0,0 +1,42 @@
+from collections.abc import Buffer
+from typing import Sequence
+
+class TileTraits:
+    """Integer arithmetics for tiles, that are arrays stored contiguously."""
+
+    def __init__(self, shape: Sequence[int, ...]): ...
+    def __repr__(self) -> str: ...
+
+    @property
+    def ndim(self) -> int: ...
+
+    @property
+    def shape(self) -> tuple[int, ...]: ...
+
+    @property
+    def stride(self) -> tuple[int, ...]: ...
+
+    @property
+    def nelems(self) -> int: ...
+
+    def linear_to_index(self, linear_offset: int) -> tuple[int, ...]: ...
+    def index_to_linear(self, linear_offset: tuple[int, ...]) -> int: ...
+
+
+class Tile(TileTraits):
+    """Many-dimensional tensor, stored contiguously in a Fortran order."""
+
+    def __init__(self, traits: TileTraits): ...
+    def from_array(self, array: Buffer): ...
+    def to_array(self, array: Buffer): ...
```

Comment:
Just like with the Tensor case, it is preferable to convert parameter array into an output

Reply:
Well, it return nothing...